Java/selectionSort.java

```diff
@@ -0,0 +1,57 @@
+import java.util.*;
+class selectionSort{
+    public static void main(String[] args){
+        int[] arr = {10,9,12,5,1,6,25};
+        /*
+        It is just like swapping the next smallest element with the present element
```

Comment:
Awesome work explaining your algorithms for beginners.

Reply:
:)